examples/linux/crash_loop_detection/crash_loop_detection.cpp

```diff
@@ -96,9 +69,44 @@ void Crash()
     memset((void*)(intptr_t)4, 123, 1);
 }
 
-int main()
+int main(int argc, char** argv)
 {
-    std::cerr << startCrashHandler() << '\n';
+    /*
+     * YOU MUST CHANGE THIS VALUE.
+     *
+     * This should point to your server dump submission port
+     * (labeled as "http/writer" in the listener configuration
+     * pane. Preferably, the SSL enabled port should
+     * be used.
+     */
+    std::string url("https://submit.backtrace.io/yolo/c5a3a1675dbcfa1d12fd7efbc7bef9350874654e55ff86478115c8bc073f54dc/minidump");
+
+    /*
+     * ENSURE THIS VALUE IS CORRECT.
+     *
+     * Crashpad has the ability to support crashes both in-process
+     * and out-of-process. The out-of-process handler is
+     * significantly more robust than traditional in-process crash
+     * handlers. This path may be relative.
+     */
+    std::string handler_path("/home/myusername/demo_crashpad_cmake/build/crashpad/handler/handler");
+
+    /*
+     * ENSURE THIS VALUE IS CORRECT.
+     *
+     * This is the directory you will use to store and
+     * queue crash data.
+     */
+    std::string db_path(".");
+
+    if (argc > 1) url = argv[1];
```

Comment:
does it make sense to assign those earlier and just reuse them instead of creating fake paths?

Reply:
This is a demo application, the fake paths also serve as an example of how the path could look like for the users of Crashpad.